src/main/resources/jupyter/init-actions.sh

```diff
@@ -103,6 +103,7 @@ if [[ "${ROLE}" == 'Master' ]]; then
     JUPYTER_USER_SCRIPT_URI=$(jupyterUserScriptUri)
     JUPYTER_USER_SCRIPT_OUTPUT_URI=$(jupyterUserScriptOutputUri)
     JUPYTER_NOTEBOOK_CONFIG_URI=$(jupyterNotebookConfigUri)
+    WELDER_ENABLED=$(welderEnabled)
```

Comment:
One thing I noticed which could maybe be fixed in this PR. Here the variable is named `WELDER_ENABLED` whereas [here](https://github.com/DataBiosphere/leonardo/blob/develop/src/main/resources/jupyter/jupyter-docker-compose.yaml#L34), [here](https://github.com/DataBiosphere/leonardo/blob/develop/src/main/resources/jupyter/jupyter_notebook_config.py#L28), and [here](https://github.com/DataBiosphere/leonardo/blob/develop/src/main/resources/jupyter/jupyter_notebook_config.py#L38) it's named `WELDER_ENABLE`. I think we need to make the variable names consistent.

Reply:
thanks for catching that!